var/spack/repos/builtin/packages/py-spglib/package.py

```diff
@@ -10,9 +10,11 @@ class PySpglib(PythonPackage):
     """Python bindings for C library for finding and handling
     crystal symmetries."""
 
-    homepage = "https://atztogo.github.io/spglib/"
+    homepage = "http://spglib.github.io/spglib"
```

Comment:
```suggestion
    homepage = "https://spglib.readthedocs.io/en/latest/"
```
This is where the link expands to. Always good to use https if possible